Cargo.toml

```diff
@@ -12,7 +12,7 @@ members = [
 default-members = ["wgpu", "player", "wgpu-hal", "wgpu-info"]
 
 [patch."https://github.com/gfx-rs/naga"]
-#naga = { path = "../naga" }
+naga = { git = "https://github.com/initial-algebra/naga", rev = "33ded1faca0fe574a81dd3dd8cb75793e6058ab9" }
```

Comment:
let's roll this back.  The way to update naga revision is by changing it in the actual dependencies, not touching the "patch" section.